contrib/crates.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+# Crates in this workspace to test (excl. fuzz an integration-tests).
+CRATES=(".")                    # Non-workspaces don't have crates.
```

Comment:
In a065c9fe4b009dc2079cbb2970a83213b602b1cb:

What is going on with this file? What does "non-workspaces don't have crates" mean, and does this actually exclude testing the `fuzz` and `integration-test` crates, and why?

Reply:
Oh yes you are correct this should include `fuzz`, and I have that wrong in `miniscript` as well. The comment is also wrong and was always wrong, I wrote "Non-workspcase" when I meant "repos that have crate root in the same directory as repo root" (I don't know the name for that). We need to keep the "." so it should be 
```
CRATES=("." "fuzz")     # Some comment about the dot directory.
```

Reply:
I think you want then

```
# Directories of each workspace to test
CRATES=("." "fuzz")
```

Saying "crates in this workspace" is just confusing. Workspaces are crates.

Reply:
BTW the reason I'm complaining about this is that the existing lockfiles don't work with MSRV, which you can tell if you try to compile the `integration_test` workspace.

Reply:
I think I also forgot that this crate is 1.48 still, at least that was not in my mind yesterday.

Reply:
oh there's the problem, I bumped the MSRV in this PR - I'll pull it out into a separate one.

Reply:
Alright, I'm well confused. I copied `Cargo-recent.lock` to `Cargo.lock` and cd'ed into `integration_test` and ran
```
 cargo +1.63.0 --locked check
```

And it was ok.

Reply:
I just went right ahead and added `fuzz` and `integration_test`  to `CRATES` also.

Note that we do not run the integration tests now or previously.

Reply:
> I think I also forgot that this crate is 1.48 still

What the hell was I thinking